### PR TITLE
Added productboard

### PR DESCRIPTION
**Pull request by chimon2000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -424,6 +424,7 @@ Table of Contents
    * https://overv.io/ - Agile project management for teams who love GitHub
    * https://taskulu.com/ - Role based project management. Free up to 5 users. Integration with GitHub/ Trello/ Dropbox/ Google Drive
    * https://www.contriber.com/ - Customizable project management platform - Free starter plan - 5 workspaces
+   * [productboard](https://www.productboard.com/) - Beautiful product management. Free for 1 editor, including Slack, Intercom, and Zendesk integrations.
```
Comment (ripienaar) [inline README.md]: looks good - but please can you make the link style as per the rest of the document?

thanks

**Comment by chimon2000** [comment]
Updated the link.

**Comment by ripienaar** [comment]
it needs a rebase as there's been many changes, ping me once you've done pls

**Comment by ripienaar** [comment]
closing, please re-open if you're still keen to get this one in and can rebase

**Pull request closed without merging**